index.js

```diff
@@ -1,15 +1,32 @@
-var FluxConstant = function FluxConstant(name) {
+var FluxConstant = function FluxConstant(name, id) {
 
-    this.name = name;
+    this.id = String(id || (FluxConstant.lastId += 1));
```

Comment:
in order for this to work, when we deserialize on the client, we will have to make sure to instantiate the constants in order, otherwise `lastId` would be incorrect.